.github/workflows/validate-translations.yml

```diff
@@ -0,0 +1,44 @@
+on:
+  pull_request:
+    paths:
+      - '**/assets/translations/**'
+
+env:
+  NODE_VERSION: '20'
+
+name: Validate Configuration of Translation Files
+jobs:
+  validate-translations:
+    name: Validate Translations
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Run Translation Validation Script
+        run: |
+          BASE_COMMIT=${{ github.event.pull_request.base.sha }}
+          HEAD_COMMIT=${{ github.event.pull_request.head.sha }}
+
+          # Get the paths of the changed files
+          changed_files=$(git diff --name-only $BASE_COMMIT $HEAD_COMMIT | grep 'assets/translations')
+
```

Comment:
Here we account only for the pattern `assets/translations`, but it can be also `assets/src/translations` (with `src` in the middle).
We need take it into account:
- in line 32
- in line 37
- in file  `ci-scripts/validate-translations.sh` in line 56